docs/wiki/installation/install-osx.md

```diff
@@ -1,3 +1,12 @@
+## System Requirements
+
+osquery has no package or library dependencies on OS X.
```

Comment:
This is an interesting statement for the top of the "install" docs. Technically... the Homebrew/custom-build install methods have quite a few dependencies. The package install has none! 

It is very valuable to communicate that the package has no dependencies, but at the same time, most packages don't have dependency lists/requirements.